CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release Notes
 
 ## 2.2.0
-- Add function for retrieving currently enabled featured  
```

Comment:
I didn't really add a new feature, just refactored the one that was already in there